setup.py

```diff
@@ -32,7 +32,7 @@
                   'rasterio': ['rasterio'],
                   'dask': ['dask>=0.16.1']}
 
-test_requires = ['rasterio', 'dask', 'xarray', 'cartopy', 'pillow', 'matplotlib', 'scipy']
+test_requires = ['rasterio', 'dask', 'xarray', 'cartopy', 'pillow', 'matplotlib', 'scipy', 'pyproj', 'numpy']
```

Comment:
`pyproj` and `numpy` are both hard requirements on the package as a whole so these shouldn't be needed here.

Reply:
This is always confusing to me. I always assume that the requirements are separate...